test/fixtures/uncaught-exceptions/uncaught-monitor2.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+// Keep the event loop alive.
+setTimeout(() => {}, 1e6);
+
+process.on('uncaughtExceptionMonitor', (err) => {
+  console.log(`Monitored: ${err.message}, will throw now`);
+  missingFunction();
+});
+
+throw new Error('Shall exit');
```

Comment:
You could turn these fixtures into message tests (the tests in tests/message that check stdio output against a .out file.)

Reply:
I thought about this but if I understand the message tests correct they assert only on the output.
Current tests verify additionally that e.g. the monitoring listener is really called so it's harder to accidentally covert them into a noop test.